meson.build

```diff
@@ -73,13 +73,13 @@ if not get_option('freetype').disabled()
   freetype_dep = dependency('freetype2', required: false)
 
   if not freetype_dep.found() and cpp.get_id() == 'msvc'
-    if cpp.has_header('ft2build.h')
-      freetype_dep = cpp.find_library('freetype', required: false)
-    endif
+    freetype_dep = cpp.find_library('freetype', required: false,
+                                    has_headers: ['ft2build.h'], )
   endif
 
-  if not freetype_dep.found() and get_option('freetype').enabled()
-    freetype_dep = dependency('freetype2', fallback: ['freetype2', 'freetype_dep'])
+  if not freetype_dep.found()
+    freetype_dep = dependency('freetype2', fallback: ['freetype2', 'freetype_dep'],
+                              default_options: ['harfbuzz=disabled'])
```

Comment:
thank you! can we not have this one implicitly?

Reply:
what do you mean?

Reply:
do we necessarily need to handle it ourselves?

> Does that make sense?

It is understandable, yes.

Reply:
Meson currently abort when a cycle is detected but I guess it could instead return not-found, that would make sense. But in the meantime I think it's still better to disable harfbuzz explicitly, I don't think it ever make sense to build freetype with harfbuzz support when we are currently building harfbuzz... does it?